doc/index.rst

```diff
@@ -190,6 +190,20 @@ Note that some of these currently tied only to a single option, but we still lea
 
 Note: You see some `*PKGS*` variables. These make things very flexible but in normal usecases you don't need to be bothered with them - just keep them blank.
 
+Use custom Docker images
+------------------------
+
+As you see in the `optional variables section <./index.rst#optional-environment-variables>`_, there are a few different ways to specify `Docker` image if you like. Here are some more detail:
+
+Pulling Docker image from an online hub
++++++++++++++++++++++++++++++++++++++++
+
+You can use `DOCKER_IMAGE` variable for this.
+
```

Comment:
Perhaps we should add a not on the requirements that such an image have to meet:
* wstool
* rosdep; including repository set-up und python-pip
* catkin-tools

Reply:
> rosdep; including repository set-up und python-pip

What is repository set-up? `rosdep init && rosdep update`?